src/psyclone/tests/dependency_test.py

```diff
@@ -234,15 +224,36 @@ def test_do_loop_not_working_yet(parser):
     assert isinstance(do_loop, nemo.NemoLoop)
     var_accesses = VariablesAccessInfo()
     do_loop.reference_accesses(var_accesses)
-    # TODO #400: n is not reported at the moment
-    assert str(var_accesses) == "ji: READWRITE, jj: READWRITE, n: READ, "\
-                                "s: WRITE, t: READ"
+    assert str(var_accesses) == "jj: READ+WRITE, n: READ, a: READ"
+
 
+def test_nemo_implicit_loop_partial(parser):
+    ''' Check the handling of ImplicitLoops access information.
+    '''
+    # TODO 440: Same as the test above but does not check the
+    # variables in the implicit loop construct, this test can
+    # be deleted when the issue is fixed and the above test
+    # passes.
+    reader = FortranStringReader('''program test_prog
+                                 do jj=1, n
+                                    s(:, jj)=t(:, jj)+a
+                                 enddo
+                                 end program test_prog''')
+    ast = parser(reader)
+    psy = PSyFactory(API).create(ast)
+    schedule = psy.invokes.get("test_prog").schedule
+
+    do_loop = schedule.children[0]
+    assert isinstance(do_loop, nemo.NemoLoop)
+    var_accesses = VariablesAccessInfo()
+    do_loop.reference_accesses(var_accesses)
+    assert str(var_accesses) == "jj: READ+WRITE, n: READ"  # a is missing
 
-@pytest.mark.xfail(reason="Gocean loops boundaries are also strings #400/#444")
+
+@pytest.mark.xfail(reason="Gocean loops boundaries are also strings #440")
```

Comment:
(Picky) but we should now drop the "also" in this reason.

Reply:
removed 'also'